apps/geo/dataloaders.py

```diff
@@ -19,7 +20,21 @@ def batch_load_fn(self, keys):
         return Promise.resolve([_map.get(key) for key in keys])
 
 
+class GeoAreaLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        ary_geo_area_qs = AssessmentRegistry.locations.through.objects\
+            .filter(assessmentregistry__in=keys).prefetch_related('geoarea')
```

Comment:
Let's use defer to remove heavy columns like polygons and centroid which are not used.